Makefile

```diff
@@ -175,6 +175,10 @@ ${RUST_MUSL_BIN}/flow-connector-init:
 ${RUST_MUSL_BIN}/flow-network-tunnel:
 	cargo build --target x86_64-unknown-linux-musl --release --locked -p network-tunnel
 
+.PHONY: ${RUST_MUSL_BIN}/flow-ops-catalog
+${RUST_MUSL_BIN}/flow-ops-catalog:
+	cargo build --target x86_64-unknown-linux-musl --release --locked -p ops-catalog
```

Comment:
fwiw I don't think this needs to be musl (it's not running inside someone else's container)

Reply:
Good call - changed to not build as musl.